### PR TITLE
[WIP] update install scripts

### PR DESCRIPTION
**Pull request by joshgav**
* Put `clean` before `install`.
* Build `subs/ai-subs` after individual subs in `install.sh`.

// ported from https://github.com/Microsoft/node-diagnostic-channel/pull/13

### CODE REVIEW

File: install.cmd [hunk 1]
```diff
@@ -4,12 +4,12 @@ setlocal enableDelayedExpansion
 
 echo Installing DiagnosticsSource
 pushd %~dp0\src\diagnostic-channel
-cmd.exe /c "npm install && npm run clean && npm test"
+cmd.exe /c "npm run clean && npm install && npm test"
```
Comment (mike-kaufman) [inline install.cmd]: Please make sure comment in other PR #13 don't get lost.

Comment (mike-kaufman) [inline install.cmd]: i.e., behavior should be consistent across .sh & .cmd scripts.

Comment (joshgav) [inline install.cmd]: will do, had to reorganize my repo after this one became public. thanks!

**Comment by mike-kaufman** [comment]
@joshgav - I believe we can close this now as, install.cmd/install.sh have been deleted & replaced w/ individual root-level tasks:

```
npm  install
npm run install-children
npm run clean
npm run build
npm run lint
npm run test
```

or alternatively
 
```
npm  install
gulp install
gulp clean
gulp build
gulp lint
gulp test
```
Please close if above commands sastisfy your intent w/ this PR.

**Comment by joshgav** [comment]
awesome, thanks Mike! once we published I didn't have to use the install scripts anymore so happy to close this.

**Pull request closed without merging**